addons/website_profile/views/website_profile.xml

```diff
@@ -0,0 +1,375 @@
+                                                <t t-else="">
+                                                    <span>0</span>
+                                                </t>
+                                            </th>
+                                        </tr>
+                                        <t t-if="user.website">
+                                            <tr>
+                                                <th style="width: 30%"><b class="text-muted">Website</b></th>
+                                                <th style="width: 70%"><span t-field="user.website"/></th>
+                                            </tr>
+                                        </t>
+                                        <t t-if="user.city or user.country_id">
+                                            <tr>
+                                                <th style="width: 30%"><b class="text-muted">Location</b></th>
+                                                <th style="width: 70%">
+                                                    <span t-field="user.city"/><t t-if="user.country_id"> (</t><span t-field="user.country_id"/><t t-if="user.country_id">)</t>
+                                                </th>
+                                            </tr>
+                                        </t>
+                                        <tr id="profile_abstract_info_company">
+                                            <th style="width: 30%"><b class="text-muted">Company</b></th>
+                                            <th style="width: 70%"><span t-field="user.company_id"/></th>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-8 mt24 ml32 o_wprofile_content_detail">
+                        <div class="tab-content mt16" id="profile_extra_info_tabcontent">
+                            <div role="tabpanel" class="tab-pane active" id="profile_tab_content_about">
+                                <div id="profile_about_badge" class="mb32">
+                                    <h4><i class="fa fa-certificate"/> Badges</h4>
+                                    <t t-call="website_profile.user_badges"></t>
+                                </div>
+                                <div t-if="user.partner_id.website_description" class="mb32">
+                                    <h4><i class="fa fa-pencil"/> Biography</h4>
+                                    <span t-field="user.partner_id.website_description"/>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </template>
+
+    <template id="profile_next_rank_card">
```

Comment:
Try to give a name to the template. I might help the support team when investigating ;)